css-grid/Overview.bs

```diff
@@ -1043,7 +1070,7 @@ Z-axis Ordering: the 'z-index' property</h3>
 	or even when positioned in non-intersecting areas because of negative margins or positioning.
 	The painting order of <a>grid items</a> is exactly the same as inline blocks [[CSS21]],
 	except that <a>order-modified document order</a> is used in place of raw document order,
-	and 'z-index' values other than <a value for=z-index>auto</a> create a stacking context even if 'position' is ''static''.
+	and 'z-index' values other than <a value for=z-index>auto</a> create a stacking context.
```

Comment:
This is a regression. 'z-index' usually doesn't apply to static boxes, so we need to specifically call this out.